doc/15-troubleshooting.md

```diff
@@ -1,5 +1,51 @@
 # Icinga 2 Troubleshooting <a id="troubleshooting"></a>
 
+## Icinga starts/restarts/reloads very slowly
+
```

Comment:
Should start with a short introduction why you might want to do this. Something along the lines that especially the startup perform many allocations so swapping out the allocator can affect the performance (both positively and negatively).